api/models/metadata.go

```diff
@@ -30,7 +29,6 @@ const (
 	labelAppName          = "gojek.com/app"
```

Comment:
Why are these not configurable using the user prefix?
```
	labelAppName          = "gojek.com/app"
	labelOrchestratorName = "gojek.com/orchestrator"
	labelEnvironment      = "gojek.com/environment"
```

Reply:
Updated